winml/adapter/winml_adapter_model.cpp

```diff
@@ -777,6 +790,139 @@ ORT_API_STATUS_IMPL(winmla::OperatorGetOutputName, _In_ const char* const op_typ
   API_IMPL_BEGIN
   auto schema = GetSchema(op_type, opset, op_domain);
   *name = schema->outputs().at(index).GetName().c_str();
+  return nullptr;
+  API_IMPL_END
+}
+
+ORT_API_STATUS_IMPL(winmla::JoinModels,
+  _In_ OrtModel* first_model,
+  _In_ OrtModel* second_model,
+  _In_ const char* const* output_names,
+  _In_ const char* const* input_names,
+  size_t num_linkages,
+  bool promote_unlinked_outputs,
+  _In_ const char* const join_node_prefix) {
+  API_IMPL_BEGIN
+
+  std::string second_model_prefix = join_node_prefix;
+  auto first_model_proto = first_model->UseModelProto();
+  auto second_model_proto = second_model->DetachModelProto();
+
+  // Remove old outputs
+  if (promote_unlinked_outputs) {
+    // loop through first model outputs and remove the linked ones in the main model inputs
```

Comment:
is this comment supposed to be connected to something?

Reply:
removed.